exporter/containerimage/attestations.go

```diff
@@ -122,7 +122,7 @@ func supplementSBOM(ctx context.Context, s session.Group, target cache.Immutable
 }
 
 func decodeSPDX(dt []byte) (s *spdx.Document, err error) {
-	doc, err := spdx_json.Load2_3(bytes.NewReader(dt))
+	doc, err := tagvalue.Read(bytes.NewReader(dt))
```

Comment:
Why does this change to tagvalue? We only produce/expect `.spdx.json`, not the tag-value format.

Reply:
@jedevc thanks! updated